deploy-agent/deployd/types/deploy_goal.py

```diff
@@ -50,6 +49,26 @@ def __init__(self, jsonValue=None):
             self.firstDeploy = jsonValue.get('firstDeploy')
             self.isDocker = jsonValue.get('isDocker')
 
+    def __eq__(self, other):
```

Comment:
why is this change?

Reply:
Change is so retry loop timers are squashed instead of reported separately.
https://github.com/pinterest/teletraan/blob/master/deploy-agent/deployd/agent.py#L369